app/views/ChangeAvatarView/index.tsx

```diff
@@ -217,6 +218,14 @@ const ChangeAvatarView = () => {
 							}
 						/>
 					) : null}
+					<Button
+						title={I18n.t('Take_a_photo')}
+						type='secondary'
+						disabled={saving}
+						backgroundColor={colors.editAndUploadButtonAvatar}
+						onPress={() => pickImage(true)}
+						testID='change-avatar-view-upload-image'
```

Comment:
```suggestion
						testID='change-avatar-view-take-a-photo'
```
The testId is conflicting with the other

Reply:
I have a doubt,I think there is no test defined for test id change-avatar-view-take-a-photo. so should I add that too?

Reply:
If you can, however it is not mandatory.